src/components/progress-fab/progress-fab.html

```diff
@@ -0,0 +1,13 @@
+<span class="md-button-wrapper">
+    <ng-content></ng-content>
+
+    <div class="md-progress-wrap">
+        <md-progress-circle

```

Comment:
To be announced, this progress circle needs to be a sibling to the focusable button, and it needs `role="progressbar"` along with other related ARIA attributes: https://www.w3.org/TR/wai-aria/roles#progressbar

Specifically the progressbar needs to specify `aria-valuenow`, `aria-valuemin` and `aria-valuemax` for screen readers to follow along. 

Lastly, what is the progress circle indicating (i.e. what is loading)? That binding should be indicated somehow...the ARIA roles spec recommends using `aria-describedby` on the progressbar and `aria-busy` on whatever is loading.